test/Microsoft.Framework.Logging.Test/LoggerMessageTest.cs

```diff
@@ -0,0 +1,204 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Framework.Logging.Test;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.Framework.Logging
+{
+    [Trait("test", "loggermessage")]
```

Comment:
Not in many places(ex: musicstore uses them), but I like them as it allows me to run only these specific tests...will remove it.